sklearn/neighbors/tests/test_neighbors.py

```diff
@@ -991,8 +990,7 @@ def test_RadiusNeighborsRegressor_multioutput(n_samples=40,
     weights = ['uniform', 'distance', _weight_func]
 
     for algorithm, weights in product(ALGORITHMS, weights):
-        rnn = neighbors.RadiusNeighborsRegressor(n_neighbors=n_neighbors,
```

Comment:
So before this was silently ignored and now it's an error? I guess that counts as bugfix ;)